CONTRIBUTING.md

```diff
@@ -1,14 +1,22 @@
 # How to contribute
 
-Operator SDK is Apache 2.0 licensed and accepts contributions via GitHub pull requests. This document outlines some of the conventions on commit message formatting, contact points for developers, and other resources to help get contributions into operator-sdk.
+Java Operator SDK is Apache 2.0 licensed and accepts contributions via GitHub pull requests. This document outlines some of the conventions on commit message formatting, contact points for developers, and other resources to help get contributions into java-operator-sdk.
+
+## Email and Chat
+
+- Email: [operator-framework][operator_framework]  
 
 ## Getting started
 
 - Fork the repository on GitHub
+    https://github.com/operator-framework/java-operator.git
+
+- If you want to build/run the project, use command
+    mvn clean install
```

Comment:
I would add a TODO or something here because we don't know the exact steps yet.

```suggestion
- If you want to build/run the project, *TBD* probably something like  use command `mvn clean install`
```